.github/actions/build-debug-turborepo/action.yml

```diff
@@ -0,0 +1,43 @@
+name: "Turborepo Build Debug"
+description: "Builds debug version of turborepo"
+inputs:
+  target:
+    description: "Compilation target"
+    required: true
+  github-token:
+    description: "GitHub token. You can pass secrets.GITHUB_TOKEN"
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: "Setup Node"
+      uses: ./.github/actions/setup-node
+    - name: "Setup Go"
+      uses: ./.github/actions/setup-go
+      with:
+        github-token: ${{ inputs.github-token }}
+    - name: "Setup Rust toolchain"
+      uses: actions-rs/toolchain@v1
+      if: ${{ inputs.target != 'windows' }}
```

Comment:
Just clarifying, if it's windows, we don't setup the rust toolchain? How does it get installed then?

Reply:
[Rustup is installed by default on GH Windows runners](https://github.com/actions/runner-images/blob/main/images/win/Windows2022-Readme.md#rust-tools). We don't use the setup action as it seems to respect the `rust-toolchain` a little too much and won't allow us to override the default host.

Here's [a log](https://github.com/vercel/turbo/actions/runs/3597814367/jobs/6060047043#step:3:266) showing that this works.